welcome/src/main/kotlin/edu/artic/welcome/WelcomeViewModel.kt

```diff
@@ -33,6 +34,7 @@ class WelcomeViewModel @Inject constructor(private val welcomePreferencesManager
                                            private val exhibitionDao: ArticExhibitionDao,
                                            generalInfoDao: GeneralInfoDao,
                                            languageSelector: LanguageSelector,
+                                           private val memberInfoPreferencesManager: MemberInfoPreferencesManager,
```

Comment:
Please move up two lines to be next to the other `private val`s.

Reply:
Done.